dist/modules/easeljs-ui.js

```diff
@@ -1690,7 +1693,7 @@
 		}
 		else
 		{
-			if (ev.pointerID != this._dragPointerID && this._dragPointerID > -1) return;
+			if (ev && ev.pointerID != this._dragPointerID && this._dragPointerID > -1) return;
```

Comment:
This function has a lot of usage of `ev`, so I think it would be better to combine it into the if statement at the top of the function, and just bail entirely if there is no event.